samplePDF/samplePDFFDBase.h

```diff
@@ -17,7 +18,6 @@
 
 #include "samplePDF/samplePDFBase.h"
 #include "samplePDF/FDMCStruct.h"
-#include "samplePDF/ShiftFunctors.h"
 
 /// @brief Class responsible for handling implementation of samples used in analysis, reweighting and returning LLH
 class samplePDFFDBase :  public samplePDFBase
```

Comment:
does it need to have FD in the name? does anything subclass `samplePDFBase` directly? Can they be merged?

Reply:
Short answer is yes for now. The ND280 code has a base class for the ND which then inherits from samplePDFBase. The goal for the next major release (or 3.0.0) is the samplePDFND object (which currently just lives in the T2K repo because it is still a bit T2K-specific) will be combined with developments we've made with the far detector code.

The reason this hasn't happened is because there are large structural differences which we're slowly trying to remove. For example the FD code has this nested structure for the oscillation channels which the ND does not care about. The ND code also keeps all samples in one samplePDFND object with just an extra dimension for each sample, making all the multi-threading etc. more efficient.

The plan is to just have a single samplePDFBase class which both ND sample and FD samples will inherit from and this is definitely what we're aiming for it just needs us to remove a few structural pathologies from the FD code.